packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -25,8 +25,15 @@ export default {
       }
     },
     stats: {
+      // 1 second intervals
       interval: 1000,
-      historyMax: 1800
+      // hold the last 2 minute of a calls data
+      historyMax: 120,
+      // should we move these to a separate config key?
+      // send a metrics event automatically
+      autoMediaMetrics: false,
```

Comment:
I think we should rename this to something like `sendDiagnostics` or `autoUploadLogs` to make it more apparent to the end user what it does.